### PR TITLE
Remove unused (and untested) parameters from private _MasterDetailFlow

### PR DESCRIPTION
**Pull request by goderbauer**
Follow-up to https://github.com/flutter/flutter/pull/126926.

### CODE REVIEW

**Comment by goderbauer** [comment]
cc @srawlins

File: packages/flutter/lib/src/material/about.dart [hunk 1]
```diff
@@ -1173,21 +1160,14 @@ class _MasterDetailFlowState extends State<_MasterDetailFlow> implements _PageOp
...
-            return _lateralUI(context);
-          } else {
-            return _nestedUI(context);
-          }
-        });
-    }
+      } else {
+        return _nestedUI(context);
+      }
+    });
```
Comment (Hixie) [inline packages/flutter/lib/src/material/about.dart:1170]: nit: avoid else after return

**Comment by Hixie** [comment]
test-exempt: removing code

File: packages/flutter/lib/src/material/about.dart (hunk 1 above)
Comment (goderbauer) [reply packages/flutter/lib/src/material/about.dart:1170]: Good call. We should probably have a lint for that...

Comment (goderbauer) [reply packages/flutter/lib/src/material/about.dart:1169]: https://github.com/dart-lang/linter/issues/4363

**Review by Piinks — APPROVED** [review]
LGTM, so nice and clean!

**Review by justinmc — APPROVED** [review]
LGTM :+1:

**Pull request merged**